include/platform/tasklet.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
The platform implements `otSignalTaskletPending`. I see this as a similar model of the alarm related functionality. But my main reason for moving it out is because it has no place in the DLL model, where a client is just calling the control APIs. They have no input to the actual worker thread loop.